Adapter/PhpcrOdmAdapter.php

```diff
@@ -137,12 +138,24 @@ public function createAutoRoute(UriContext $uriContext, $contentDocument, $autoR
             $parentDocument = $document;
         }
 
+        $finalAutoRoutePath = $path . '/' . $headName;
+        $node = $this->dm->find(null, $finalAutoRoutePath);
```

Comment:
rename `$node` to `$document`. We generally use `node` to refer to PHPCR nodes, and `document` to refer to the objects returned from the document manager.